neuralpp/symbolic/sympy_expression.py

```diff
@@ -484,6 +553,34 @@ def native_arguments(self) -> Tuple[sympy.Basic, ...]:
         return sympy_piecewise_to_if_then_else(self.sympy_object)
 
 
+class SymPyPiecewise(SymPyFunctionApplicationInterface):
```

Comment:
new class `SymPyPiecewise`.

Note that `piecewise` in SymPy has its arguments as `[(expression1, condition1), (expression2, condition2), ...]` (each element is a tuple) where as arguments of our `SymPyPiecewise` is `[expression1, condition1, expression2, condition2]` (each element is an `Expression`, which is consistent with the our library interface).